src/core/Expander/Expander/Expander.tsx

```diff
@@ -0,0 +1,226 @@
+import React, { Component, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+import { AutoId } from '../../../utils/AutoId';
+import { TokensProp, InternalTokensProp } from '../../theme';
+import { HtmlDiv } from '../../../reset';
+import { baseStyles } from './Expander.baseStyles';
+import {
+  ExpanderGroupConsumer,
+  ExpanderGroupProviderState,
+} from '../ExpanderGroup/ExpanderGroup';
+
+const baseClassName = 'fi-expander';
+const openClassName = `${baseClassName}--open`;
+
+export interface ExpanderProviderState {
+  /** Callback for communicating ExpanderTitle button event to Expander  */
+  onToggleExpander: () => void;
+  /** Open state for expander */
+  open: boolean;
+  /** Id for expander button */
+  titleId: string | undefined;
+  /** Id for expander content */
+  contentId: string | undefined;
+}
+
+const defaultProviderValue: ExpanderProviderState = {
+  onToggleExpander: () => null,
+  open: false,
+  titleId: undefined,
+  contentId: undefined,
+};
+
+const {
+  Provider: ExpanderProvider,
+  Consumer: ExpanderConsumer,
+} = React.createContext(defaultProviderValue);
+
+interface InternalExpanderProps {
+  /**
+   * Children, extend type ExpanderTitleBaseProps or ExpanderContentBaseProps
+   * ExpanderProviderState context is used to communicate between title, content and expander
+   */
+  children: ReactNode;
+  /** Custom classname to extend or customize */
+  className?: string;
+  /**
+   * Id for expander, must be unique. Duplicate id's break ExpanderGroup functionality.
+   * Autogenerated if not provided
+   */
+  id?: string;
+  /** Default status of expander open
```

Comment:
> Expander is opened by default

Would this be better? As the possible values are `true/false`

Reply:
Expanders are actually closed by default and this can change it. Would be good to come up with better wording though... "Expanders initial open state" with current @default false?

Reply:
Yes, they are closed by default and tried to describe the situation where the boolean answers it when reading it
> Expander is opened by default: false

 as the false is default value in this case.

But the suggested is better:
> Expanders initial open state

Reply:
Changed it to "Initial open state" since repeating Expander didn't feel necessary.